composer.json

```diff
@@ -24,6 +24,6 @@
         "symfony/security-csrf": "~2.5",
         "symfony/twig-bridge":   "~2.5",
         "org_heigl/daterange": "~1.1",
-        "guzzlehttp/guzzle": "^6.1"
+        "guzzlehttp/guzzle": "~6.2.1"
```

Comment:
I think that the `~` in the other options indicates when they were added and for a `major.minor` constraint it doesn't make any practical difference if you use `~` or `^`. However if you use a `major.minor.patch` constraint, then it does make a difference and `^` is preferred.